CONTRIBUTING.md

```diff
@@ -34,6 +34,58 @@ $ grunt docs
 $ grunt live
 ```
 
-When using the server provided by `grunt live`, the Strand documentation can be accessed at http://localhost:8000/docs/
+When using the server provided by `grunt live`, the Strand documentation can be accessed at `http://localhost:8000/docs/`
+
+## <a name="issue"></a> Found an Issue?
+If you find a bug in the source code or a mistake in the documentation, help us by
+submitting an [issue](https://github.com/MediaMath/strand/issues). If you have a solution, you can submit a [Pull Request](#pr) with the fix.
+
+## <a name="pr"></a>Submitting a Pull Request
+Before you submit your pull request consider the following guidelines:
+
+* Search [GitHub](https://github.com/MediaMath/strand/pulls) for an open or closed Pull Request
+  that relates to your submission. You don't want to duplicate effort.
+* Create a new branch from `develop`
+* Make your changes in a new git branch:
+
+     ```
+     git checkout -b my-branch-name
+     ```
+
+* Create your patch
+* Follow our [Coding Guidelines](#guidelines).
+* Commit your changes using a descriptive commit message
+* Push your branch to GitHub:
+
+    ```
+    git push origin develop my-branch-name
+    ```
+
+* Squash your commits, if there are many
+* In GitHub, send a pull request to `strand:develop`.
+* If we suggest changes then:
+  * Make the required updates
+  * Rebase your branch and force push to your GitHub repository (this will update your Pull Request):
+
```

Comment:
maybe something about fetching upstream changes here..

should probably do something like `git fetch upstream`
and then `git rebase -i upstream/develop`